src/extend.js

```diff
@@ -0,0 +1,3 @@
+const extend = (x = {}, y = {}) => Object.assign({}, x, y)
+
+module.exports = extend
```

Comment:
Could we change this file to `utils.js` and use `exports.extends` instead? Seems fairly likely we'd want to add other utils there in the future

Reply:
Done